### DISCUSSION TITLE
Missing SDF collisions in CPU mode

### DISCUSSION THREAD

**Discussion by proc-sim** [NVIDIA-Omniverse/PhysX#98] [Q&A] [answered]
If I setup a dynamic rigidbody with an SDF collider (using settings similar to those in the SDF snippet), I get expected results when simulating on the GPU (collisions between dynamic SDF and static ground collider, collisions between multiple dynamic SDF meshes).

However, if I simulate on the CPU, the SDF colliders will collide with kinematic rigidbodies and rigidbodies with box/sphere/convex hulls, but won't collide other rigidbodies with SDF hulls and won't collide with static colliders.

**Comment by preist-nvidia** [comment, marked as answer]
Hi @tysoni - SDF collision is a GPU only feature, so it does not work with CPU simulation.

**Comment by proc-sim** [comment]
Well, it works partially on CPU because my dynamic SDF rigidbodies collide correctly with kinematic rigidbodies in CPU mode, but it's good to know that the rest of the incorrect behavior is expected. 

Is full CPU support expected in the future?

**Reply by preist-nvidia** [reply]
Yes, there is a fallback, but it will not use SDF collision.

It is currently not planned to add CPU support for the SDF feature.

**Reply by proc-sim** [reply]
Ah, it looked like some proper concave collisions were happening in CPU mode, so that's why I assumed it was partially working. Thanks for the clarification.
